test/unit/Exception/CannotParseOpenApiTest.php

```diff
@@ -0,0 +1,167 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OnMoon\OpenApiServerBundle\Test\Unit\Exception;
+
+use OnMoon\OpenApiServerBundle\Exception\CannotParseOpenApi;
+use PHPUnit\Framework\TestCase;
+
+use function Safe\sprintf;
+
+/**
+ * @covers \OnMoon\OpenApiServerBundle\Exception\CannotParseOpenApi
+ */
+final class CannotParseOpenApiTest extends TestCase
+{
+    public function tesBecauseNoOperationIdSpecifiedShowsCorrectErrorMessage(): void
```

Comment:
This method is not executed because of a typo in the method name.
Please check coverage report
https://coveralls.io/builds/32500345/source?filename=src/Exception/CannotParseOpenApi.php